### DISCUSSION TITLE
Some numbers I've been given - anonymous

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/BurnoutRecoveryIndex#2] [General]
My current burnout score is 73

A few others: 

58 - corporate job, recovering from burnout, looking forward to quitting
44 - someone who actively hates their job 

A few similar scores around.

**Comment by se5a** [comment]
I got through the mental health sections before I got bored and generated the score.
My aggregate score at that point was 41.56 / 100, and considering I just handed in my notice yesterday, your 44 for someone who actively hates their job sounds about right.
Annoyingly, I loved the work, just hated the job. *shrug*
